samples/bootloader/README.rst

```diff
@@ -72,7 +72,7 @@ OTP regions
 
 The one-time programmable (OTP) region is a special region of the *User Information Configuration Registers* (UICR) that only allows flash memory writes in half-word lengths, and *only* when the target half-word has the value of ``0xFFFF``.
 
-On the SoCs that support an OTP region, such as the nRF9160 and nRF5340, the provisioned data is held in the OTP region instead of the internal flash memory.
+On the SoCs that support an OTP region, such as the nRF91 Series and nRF5340, the provisioned data is held in the OTP region instead of the internal flash memory.
```

Comment:
```suggestion
On products that support an OTP region, such as the nRF91 Series and nRF5340, the provisioned data is held in the OTP region instead of the internal flash memory.
```

Reply:
As I mentioned, the nRF91 Series products are not SoCs. I must have overlooked this text in the last round.